app/models/theme.js

```diff
@@ -1,15 +1,10 @@
 import Model, { hasMany, attr } from '@ember-data/model';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  label: attr('string'),
-  scopeNote: attr('string'),
-  subcase: hasMany('subcase', {
-    inverse: null,
-  }),
-  agendaitem: hasMany('subcase', {
-    inverse: null,
-  }),
-  deprecated: attr('boolean'),
-});
+export default class Theme extends Model {
+  @attr('string') label;
+  @attr('string') scopeNote;
+  @attr('boolean') deprecated;
+
+  @hasMany('subcase', { inverse: null }) subcase;
```

Comment:
subcase doesn't have themes, neither does agendaitem (themes used to be on here, maybe some legacy data remains in db)
Newsletter-info does, but there is no point to mention this relation here since we should never ask "what newsletters have this theme"  by doing `theme.newsletters`. We would use a query in that scenario.